content/ri/memory-optimizations.md

```diff
@@ -69,7 +69,6 @@ You can follow one of these three steps to reclaim the memory:
 1. Restart your redis-server
 1. Increase memorysamples in redis conf. (default is 5, max is 10) so that expired keys are reclaimed faster.
```

Comment:
```suggestion
1. Increase 'maxmemory-samples` in redis.conf (default is 5, max is 10) so that the expired keys are reclaimed faster.
```

Reply:
'(here should be \` )maxmemory-samples`.

Reply:
You're right. Thanks!

Reply:
`max is 10` is wrong, 10 only means better accuracy, but will consume more cpu, its max value is int_max.